src/parse/asp/objects.go

```diff
@@ -964,6 +964,9 @@ func loadPluginConfig(subrepoConfig *core.Configuration, packageState *core.Buil
 	contextPackage := &core.Package{SubrepoName: packageState.CurrentSubrepo}
 	configValueDefinitions := subrepoConfig.PluginConfig
 	for key, definition := range configValueDefinitions {
+		if definition.ConfigKey == "" {
+			definition.ConfigKey = strings.ReplaceAll(key, "_", "")
+		}
```

Comment:
We don't want to write this back to the definition do we? Probably just want to create a local variable for it.

Reply:
Oh sorry thought we did! Updated it now.